src/server/routes/comments.js

```diff
@@ -20,26 +20,27 @@ router.post("/:id", (req, res) => {
     .catch(err => res.json(err));
 });
 
-router.delete("/:id/comments/", (req, res) => {
-  const id = req.body;
-
-  // console.log(id._id);
-  Card.findById(req.params.id).then(card => {
-    // const list = card.comments.filter(comment => {
-    //   console.log("***");
-    //   console.log(comment._id);
-    //   console.log(id._id);
-    //   console.log("***");
-    //   return comment._id !== id._id;
-    // });
-    const index = card.comments
-      .map(comment => {
-        return comment._id;
-      })
-      .indexOf(id._id);
-    card.comments.splice(index, 1);
-    console.log(card.comments);
+router.patch("/:id/comments/:comment_id", (req, res) => {
+  Card.findByIdAndUpdate(req.params.id).then(card => {
+    const editedComment = card.comments.filter(comment => {
+      return comment._id.toString() === req.params.comment_id;
+    });
+    editedComment[0].comment = req.body.comment;
+    console.log(editedComment);
+    card.save().then(res.json(card));
```

Comment:
Shouldn't it be `card.save().then(() => res.json.(card))`? Because `res.json(card)` is a function call and not a function.